src/lib/badge/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+package(default_visibility=["//visibility:public"])
+load("@angular//:index.bzl", "ng_module")
+load("@io_bazel_rules_sass//sass:sass.bzl", "sass_library", "sass_binary")
```

Comment:
you might want to hook up skylint to tell you about unused imports and such

and buildifier too

Reply:
One of the tasks I have tracked in #8369